mscore/musescore.cpp

```diff
@@ -7110,6 +7110,9 @@ void tryToRequestTelemetryPermission()
       {
       QString accessRequestedAtVersion = preferences.getString(PREF_APP_STARTUP_TELEMETRY_ACCESS_REQUESTED);
 
+      if (MScore::noGui))
+            return;
+
```

Comment:
I guess/hope this does disable telemetry itself for the duration of that `noGui` session too?

Reply:
It is disabled by default so if user didn't enable it first in some GUI session it will not enable itself. The question arises for the case if user has enabled telemetry and then upgraded to a version where we decide to show telemetry dialog again, but for that case we need to decide something on the topic of #5618 first.

Reply:
it needs to default to no in that case too, for that session, but ask again next time in GUI mode